pricelist_per_product/README.rst

```diff
@@ -0,0 +1,59 @@
+Pricelist Per Product
+======================
+
+This module allows to see all sale prices on product form
```

Comment:
suggestion: s/to see/to see and edit/

s/on product form/on the product form./